cs/test/core/Core.csproj

```diff
@@ -120,7 +79,7 @@
     </Reference>
   </ItemGroup>
   <ItemGroup>
-    <None Include="packages.config" />
+    <Service Include="{82a7f48d-3b50-4b1e-b82e-3ada8210c358}" />
```

Comment:
What does a `Service` item do?

Reply:
No idea how that got there. I'll double check.